pyperformance/benchmarks/bm_tornado_http.py

```diff
@@ -86,6 +87,11 @@ def run_client():
 
 
 if __name__ == "__main__":
+    # https://github.com/tornadoweb/tornado/pull/2686
+    if sys.platform == 'win32' and sys.version_info[0] != 2:
```

Comment:
I suggest sys.version_info[:2] == (3, 8) test instead. I want to believe that https://bugs.python.org/issue37373 will be implemented in Python 3.9. If not, it will be easy to update this test.